### DISCUSSION TITLE
Should we connect with apicuron as a mechanism for giving credit for ontologies?

### DISCUSSION THREAD

**Discussion by cmungall** [OBOFoundry/OBOFoundry.github.io#2350] [Ideas]
https://apicuron.org/

Example of how this might work:
https://apicuron.org/databases/bioregistry

**Comment by matentzn** [comment]
I am thinking about this as well. I don't think the current curation models at apicuron would work for us (pushing using the API, manual upload), but here is one way that could work:

1. We use the format proposed by apicuron form capturing curation activities with attribution: https://apicuron.org/docs#bulk-submission (personally I would request "activity_term" to be a proper identifier, so that it is more flexible moving forward).
   ```
   {
   "activity_term": "region_first_generation",
   "resource_id": "MONDO",
   "timestamp": "2016-08-11T09:59:17.000Z",
   "curator_orcid": "0000-0002-0341-4888",
   "entity_uri": "http://purl.obolibrary.org/obo/MONDO_123"
   }
  ```
2. All ontologies publish a "curation attribution" file as part of their ontology release process.
3. We collect all of these at OBO foundry (if "http://purl.obolibrary.org/obo/cl/attribution.json" exists ...) or register them individually at apicuron (1 project per ontology, 1 huge file per OBO Foundry)
4. We talk to APICURON to pull this file from a PURL rather than us pushing it.

**Comment by matentzn** [comment]
We should discuss befor going all in on APICURON what our specific use cases are. Quantifying curation effort is one thing, but what exactly is it that we want?

1. Badges? For what?
2. Documented activity on ORCID profile pages? 
3. Wikidata dashboards?

We have not really thought this whole thing through to the very end yet - I would like to avoid something like an emphasis on rankings, and focus more on individual attributions. And I am not sure I like this idea of "newbie" and "bronze" badges yet. I am wondering where this should be discussed in more detail. The CV use case presented as biocuration is one that makes sense, but maybe there is more.
